pytest_picked/plugin.py

```diff
@@ -41,10 +41,15 @@ def _get_affected_paths(config):
     picked_mode = config.getoption("picked_mode")
     parent_branch = config.getoption("parent_branch")
     test_file_convention = config._getini("python_files")  # pylint: disable=W0212
+    root_dir = str(config.rootdir) + "/"
```

Comment:
@anapaulagomes let me know if you prefer an "f" string here, instead.

Reply:
I do :)

Reply:
Fixed